src/mango/rebar.config.script

```diff
@@ -12,16 +12,16 @@
 
 DreyfusAppFile  = filename:join(filename:dirname(SCRIPT),
     "../dreyfus/src/dreyfus.app.src"),
-RenameFile = filename:join(filename:dirname(SCRIPT),
-    "src/mango_cursor_text.erl"),
```

Comment:
And there's a weird breakage down below with a file rename that looks not right to me.